core/src/main/java/tc/oc/pgm/listeners/PGMListener.java

```diff
@@ -258,6 +261,30 @@ public void handleItemPickup(final PlayerPickupItemEvent event) {
     if (nearestPlayer != event.getPlayer()) event.setCancelled(true);
   }
 
+  @EventHandler
+  public void lockFireTick(final MatchLoadEvent event) {
+    oldFireTick = Boolean.parseBoolean(event.getMatch().getWorld().getGameRuleValue(DO_FIRE_TICK));
+    event.getMatch().getWorld().setGameRuleValue(DO_FIRE_TICK, Boolean.toString(false));
+  }
+
+  // if no doFireTick <gamerule> is found, then use doFireTick setting saved in world
+  @EventHandler
+  public void unlockFireTick(final MatchStartEvent event) {
+    ImmutableMap<String, String> gameRules =
+        event.getMatch().getModule(GameRulesMatchModule.class).getGameRules();
+    String newFireTick = gameRules.get(DO_FIRE_TICK);
```

Comment:
you're doing a `get` here just to later ignore the result if is empty, bit of a weird one, could simply `getOrDefault` to oldFireTick and simplify this further